guides/source/active_support_core_extensions.md

```diff
@@ -151,8 +151,8 @@ NOTE: Defined in `active_support/core_ext/object/blank.rb`.
 As of Ruby 2.5, most objects can be duplicated via `dup` or `clone`:
 
 ```ruby
-"foo".dup           # => "foo"
-"".dup              # => ""
++"foo"           # => "foo"
++""              # => ""
```

Comment:
I understand that this follows rubocop-rails-omakase, but this change will likely become irrelevant in the near future. I have opened a PR regarding this concern in rubocop-rails-omakase.
https://github.com/rails/rubocop-rails-omakase/pull/22

Even if using `String#+@`, the indentation should be aligned.

```suggestion
+"foo"              # => "foo"
+""                 # => ""
```

Reply:
Great catch. Even so, the guide talks about using `dup` so it should continue to use that.

Unfortunatly inline disabling of cops with `rubocop-md` is broken right now: https://github.com/rubocop/rubocop-md/pull/36.

I wonder if performance cops should be part of the guides in the first place. I understand what this PR is for but they don't always produce more pleasant code. When doing `inherit_from: '../.rubocop.yml'`, that should happen automatically (at least for this specific instance) since there are already some excludes specific to the guides in there.

Reply:
Ok, looks like that was merged. Should we wait for a release?

This is before using git source:

```
3321 files inspected, 459 offenses detected, 261 offenses autocorrectable
```

And with the latest changes:

```
3321 files inspected, 457 offenses detected, 259 offenses autocorrectable
```

This is with `Layout/SpaceInsideArrayLiteralBrackets` disabled:

```
3321 files inspected, 61 offenses detected, 61 offenses autocorrectable
```

So we could almost split that into separate PRs to make it easier to review. :thinking:

Reply:
> I wonder if performance cops should be part of the guides in the first place

Yeah, my first instinct is that all performance cops should be disabled for documentation, for similar reasons that they are for Rails' own tests: nudging spacing around is one thing, but if we're trying to document a particular method, it's unhelpful to [auto- or otherwise] "correct" the example to do something different.

On the other hand, _outside_ of examples where we might be referencing a very specific method (as in this case), or giving a long-form example to describe what the "better" spelling would be equivalent to, it probably is reasonable that we'd want to at least _know_ that we're giving people code that will fail default linting.

So if we can have inline disabling, maybe that is good enough? Fortunately it doesn't seem to have come up much / at all here so far, apart from this one.

---

> looks like that was merged. Should we wait for a release?

Will this just work if we point at the Git source? Probably makes more sense than depending on a release of our own inter-related library, even if it's not expected to change often. 🤷🏻‍♂️